src/app_service/service/community/service.nim

```diff
@@ -665,8 +665,11 @@ QtObject:
     return toSeq(self.getFilteredCuratedCommunities.values)
 
   proc getCommunityById*(self: Service, communityId: string): CommunityDto =
-    if(not self.communities.hasKey(communityId)):
-      error "error: requested community doesn't exists", communityId
+    if communityId == "":
```

Comment:
Maybe we have a deeper issue here? Why are we calling `getCommunityById` with an empty string? It's probably nothing, but it would be nice to know.
Not that it needs to be fixed in this PR, it's just that this change will hide the symptom so we'll forget haha